src/Uno.Foundation/Rect.Android.cs

```diff
@@ -12,6 +12,10 @@ public partial struct Rect
 		public static implicit operator Rect(Android.Graphics.Rect rect) => new Rect(rect.Left, rect.Top, rect.Width(), rect.Height());
 
 		public static implicit operator Android.Graphics.Rect(Rect rect) => new Android.Graphics.Rect((int)rect.X, (int)rect.Y, (int)(rect.X + rect.Width), (int)(rect.Y + rect.Height));
+
+		public static implicit operator Rect(Android.Graphics.RectF rect) => new Rect(rect.Left, rect.Top, rect.Width(), rect.Height());
+
+		public static implicit operator Android.Graphics.RectF(Rect rect) => new Android.Graphics.RectF((int)rect.X, (int)rect.Y, (int)(rect.X + rect.Width), (int)(rect.Y + rect.Height));
```

Comment:
I'm wondering if we should support equality between Rect and RectF when we're using implicit operators.